packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -940,6 +971,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     private addContainerBlobsToSummary(summaryTree: ISummaryTreeWithStats) {
+        if (!this.runtimeOptions.disableIsolatedChannels) {
+            const metadata: IContainerRuntimeMetadata = { snapshotFormatVersion: 1 };
+            addBlobToSummary(summaryTree, metadataBlobName, JSON.stringify(metadata));
```

Comment:
Should we call this blob ".attributes" to be consistent with other layers?

Reply:
We can, but it's `.component` in the datastore layer, and personally prefer metadata over attributes in terms of terminology. But happy to make a change for consistency if you think it makes more sense.

Reply:
I think I saw a different .attributes in the root at some point when debugging? Feels like it might be risky. I think it may have been somewhere we use the protocol .attributes blob in the root, but I'm not 100% certain.

Reply:
It's fine if its risky. I don't think I care that much. The .component is the one that bugs me more :)